Classes/market.py

```diff
@@ -0,0 +1,10 @@
+class Market:
+  def __init__(self, cards):
+    self.active_market = cards[:4]
+    self.inactive_market = cards[4:]
+  
+  def purchase_card(self, index_purchased_card, card_to_add):
```

Comment:
Here's where we can put useful logic for the market.  So anytime a player buys from the market, we'll remove the card, insert the new one, and then reorganize the market.  I'm sure we'll have many more functions in here to manipulate the market